src/gc/env/gcenv.base.h

```diff
@@ -551,6 +551,7 @@ void LogSpewAlways(const char *fmt, ...);
 void StompWriteBarrierEphemeral(bool isRuntimeSuspended);
 void StompWriteBarrierResize(bool isRuntimeSuspended, bool bReqUpperBoundsCheck);
 bool IsSuspendEEThread();
```

Comment:
I do not think there is going to be integration from release branch back to master branch.